rhizome/host/lib/cloud_hypervisor.rb

```diff
@@ -13,10 +14,38 @@ def path
     end
   }
 
+  FirmwareClass = Struct.new(:arch, :version) {
+    def url
+      "https://github.com/ubicloud/build-edk2-firmware/releases/download/edk2-stable#{version}-#{arch}/CLOUDHV-#{arch}.fd"
+    end
+
+    def firmware_root
+      "/opt/fw"
+    end
+
+    def path
+      "#{firmware_root}/CLOUDHV-#{arch}-#{version}.fd"
```

Comment:
Since the VM host's architecture doesn't change and it's a single architecture, should we keep `arch` in the firmware name? Why not just use `"#{firmware_root}/CLOUDHV-#{version}.fd"` similar to our boot image names?

Reply:
Yeah, there is no great reason that I know of yet, since we'll won't have multiple architectures present simultaneously.

Reply:
I also went back and forth between these two alternatives. In the latest iteration, we omit the architecture from the filename. We have one more safeguard in place as we now check the firmware's digest. But I'm happy to add it back if there are concerns.